jdaviz/configs/cubeviz/helper.py

```diff
@@ -51,6 +55,12 @@ def load_data(self, data, data_label=None, **kwargs):
             raise RuntimeError('only one cube may be loaded per Cubeviz instance')
         if data_label:
             kwargs['data_label'] = data_label
+
+        kwargs.update({
+            "flux_viewer_reference_name": self._default_flux_viewer_reference_name,
+            "uncert_viewer_reference_name": self._default_uncert_viewer_reference_name,
+            "spectrum_viewer_reference_name": self._default_spectrum_viewer_reference_name,
+        })
```

Comment:
Is there a reason why these passed here as `kwargs` instead of accessed directly in `load_data`?  If we want users to be able to override the values passed to `load_data`, then maybe we want `kwargs.setdefault` instead?